src/index.js

```diff
@@ -1 +1,40 @@
-import './index.css';
\ No newline at end of file
+import './index.css';
+import {
+  apiServer, createScore, fetchScore, addScore,
+} from './modules/api.js';
+
+let id = '';
+const refreshBtn = document.querySelector('.refresh-btn');
+const addBtn = document.querySelector('.add-btn');
+
+const startGame = async () => {
+  const response = await fetch(`${apiServer}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({ name: 'Leaderboard' }),
+  });
+  const data = await response.json();
+  id = data.result;
+};
+
+startGame();
+
```

Comment:
- [ ] It would be nice if you have a hard code id in your code instead of creating an id every time when your app reloads because when your app reloads then it creates a new id means your API is changed and your previous data is no more exist 😨 Kindly fix this issue to make your app have the data even it reloads thousand times 👍